pcsx2/gui/MainFrame.cpp

```diff
@@ -402,9 +402,12 @@ void MainEmuFrame::CreatePcsx2Menu()
 								 wxEmptyString, wxITEM_CHECK);
 
 	m_menuSys.AppendSeparator();
-
-	m_menuSys.Append(MenuId_Sys_LoadStates, _("&Load state"), &m_LoadStatesSubmenu);
-	m_menuSys.Append(MenuId_Sys_SaveStates, _("&Save state"), &m_SaveStatesSubmenu);
+	//  Implement custom hotkeys (F3) with translatable string intact + not blank in GUI.
+	wxMenuItem* MainLoadStateLabel = m_menuSys.Append(MenuId_Sys_LoadStates, _("&Load state"), &m_LoadStatesSubmenu);
```

Comment:
nit (and similar below): I'm not a fan of this variable naming.  The value of `MainLoadStateLabel` isn't really a label, it's a menu item.  Perhaps `sysLoadStateMenuItem` would be better?  I'm also not a fan of the uppercase on the first character, but I might be contradicting the styleguide here.

Reply:
Fair enough, i've also changed the wxMenuItems for Capture and Screenshot.